lib/matplotlib/tests/test_figure.py

```diff
@@ -1770,6 +1770,44 @@ def test_warn_colorbar_mismatch():
         subfig3_1.colorbar(im4_1)
 
 
+def test_clf_subplotpars():
+    keys = ('left', 'right', 'bottom', 'top', 'wspace', 'hspace')
+    rc_params = {key: plt.rcParams['figure.subplot.'+key] for key in keys}
+
+    fig = plt.figure(1)
+    fig.subplots_adjust(left=0.1)
```

Comment:
I suggest to modify all parameters for better coverage.

```suggestion
    fig.subplots_adjust({k, v+0.01 for k, v in rc_params.items())
```